_opensearch/rest-api/ingest-apis/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Ingest APIs
+parent: REST API reference
+has_children: true
+nav_order: 3
+redirect_from:
+  - /opensearch/rest-api/ingest-apis/
+---
+
+# Ingest APIs
+
+Before you index your data, OpenSearch's ingest APIs help transform your data through the creation of ingest pipelines. Pipelines are made up of processors, a customizable task that run in the order they appear in the request body. The transformed data appears in your data stream or index after each of the processors completes.
```

Comment:
"processors, which are customizable **tasks** that run in the order they appear..."